packages/gatsby-source-contentful/src/gatsby-node.js

```diff
@@ -167,30 +213,42 @@ exports.sourceNodes = async (
         })
       }
 
-      const entryNode = {
+      let entryNode = {
         id: entryItem.sys.id,
         parent: contentTypeItemId,
         children: [],
-        ...entryItemFields,
         internal: {
           type: `${makeTypeName(contentTypeItemId)}`,
-          content: JSON.stringify(entryItem),
           mediaType: `application/json`,
         },
       }
 
+      // Replace text fields with text nodes so we can process their markdown
```

Comment:
Nice, in a separate project I ended up doing...

`<div dangerouslySetInnerHTML={{ __html: md.render(content) }} />`

... with a text field using `MarkdownIt`. Having built in support will be nice. I can update the `using-contentful` site later to exercise this functionality.